deepcell_toolbox/utils.py

```diff
@@ -130,7 +130,7 @@ def _shift_image(img, shift):
     return X
 
 
-def tile_image(image, model_input_shape=(512, 512), stride_ratio=0.75):
+def tile_image_deprecated(image, model_input_shape=(512, 512), stride_ratio=0.66):
```

Comment:
Le'ts change back the default `stride_ratio`

Reply:
Done. I just changed it when I was testing to make sure the padding worked.